### PR TITLE
feat:  add html response

### PR DESCRIPTION
**Pull request by 2264381489**
add html response

### CODE REVIEW

**Comment by 2264381489** [comment]
todo: add test

File: http/responses.go [hunk 1]
```diff
@@ -56,3 +57,52 @@ func doWriteXml(w http.ResponseWriter, code int, v any) error {
...
+}
+
+func doWriteHTML(w http.ResponseWriter, code int, v any) error {
+	if reflect.TypeOf(v).Kind() != reflect.String {
+		err := fmt.Errorf("the value must be a string, but got: %v", v)
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return err
+	}
+
+	bs := v.(string)
```
Comment (kevwan) [inline http/responses.go]: reflect kind is String not means the real type is string.

For example, `type MyString string`

Comment (2264381489) [reply http/responses.go]: fix

Comment (2264381489) [reply http/responses.go]: @kevwan ke fix

File: http/responses.go [hunk 2]
```diff
@@ -56,3 +55,54 @@ func doWriteXml(w http.ResponseWriter, code int, v any) error {
...
+}
+
+// WriteHTMLCtx writes v as HTML string into w with code.
+func WriteHTMLCtx(ctx context.Context, w http.ResponseWriter, code int, v any) {
+	if err := doWriteHTML(w, code, v); err != nil {
+		logx.WithContext(ctx).Error(err)
+	}
+}
+
+func doWriteHTML(w http.ResponseWriter, code int, v any) error {
```
Comment (kesonan) [inline http/responses.go]: Why not show that the definition type is string?

Comment (2264381489) [reply http/responses.go:83]: good idea

Comment (2264381489) [reply http/responses.go]: @kesonan FIX

Comment (2264381489) [reply http/responses.go]: @kesonan ken fix

**Review by kesonan — APPROVED** [review]
LGTM

**Comment by 2264381489** [comment]
thanks

**Pull request merged**